element_animal/genotyping.py

```diff
@@ -14,27 +14,29 @@ def activate(
     create_tables=True,
     linking_module=None,
 ):
-    """
-    activate(genotyping_schema_name, subject_schema_name=None,
-             create_schema=True, create_tables=True, linking_module=None)
-        :param genotyping_schema_name: schema name on the database server to
-                                       activate the `genotyping` element
-        :param subject_schema_name: schema name on the database server to
-                                    activate the `subject` element
-        :param create_schema: when True (default), create schema in the
-                              database if it does not yet exist.
-        :param create_tables: when True (default), create tables in the
-                              database if they do not yet exist.
-        :param linking_module: a module name or a module containing the
-         required dependencies to activate the `genotyping` element:
-             Upstream tables:
-                + Source: the source of the material/resources
-                          (e.g. allele, animal) - typically refers to the
-                          vendor (e.g. Jackson Lab - JAX)
-                + Lab: the lab for which a particular animal belongs to
-                + Protocol: the protocol applicable to a particular animal
-                            (e.g. IACUC, IRB)
-                + User: the user associated with a particular animal
+    """Activate this schema.
+
+    Args:
+        genotyping_schema_name (str): schema name on the database server to
+                                    activate the `genotyping` element.
+        subject_schema_name (str): schema name on the database server to
+                                activate the `subject` element
+        create_schema (bool, optional): when True (default), create schema in the
+                            database if it does not yet exist.
+        create_tables (bool, optional): when True (default), create tables in the
+                            database if they do not yet exist.
+        linking_module (bool, optional): a module name or a module containing the
+        required dependencies to activate the `subject` element:
+
+    Note:
+            Upstream tables:
+            + Source: the source of the material/resources
```

Comment:
Does this render the line breaks in local deployment? I managed to do it for the functions, but not dependencies.
- [deployed site](https://datajoint.com/docs/elements/element-deeplabcut/0.2/api/element_deeplabcut/model/)
- [docstring](https://github.com/datajoint/element-deeplabcut/blob/main/element_deeplabcut/model.py#L44-L52)

Reply:
It doesn't. I've changed it to follow the formatting from other repos